spec/models/announcement_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe Announcement, type: :model do
+  ANNOUNCEMENT_VALID_PARAMS = { episode_id: 1, title: 'Title', date: Time.now, target_resource: 'telegram' }.freeze
```

Comment:
Check this for testing validation https://github.com/thoughtbot/shoulda-matchers